swift-extensions/UISwitchExtensions.swift

```diff
@@ -19,6 +19,8 @@ extension UISwitch {
             observe(toggleSwitchViewModel.isOn) { [weak self] (value: Bool) in
                 self?.isOn = value
             }
+
+            bind(toggleSwitchViewModel.enabled, \UISwitch.isEnabled)
```

Comment:
Following [my comment](https://github.com/mirego/trikot.viewmodels/pull/72#discussion_r569471817) on the other related PR, this could be done on the UIControl class instead (if we chose to go forward with ControlViewModel)

Reply:
I would also remove action from ViewModel but that's a breaking change. To be discussed

Reply:
I agree with moving the action too. Let's keep that as a future improvement 👍